File: 1812/Domain/Commander.cs
namespace _1812.Domain
{

    public class Commander
    {
        public string ID { get; }
        public string Name { get; }
        public string Description { get; }
        public string ImageFolder { get; }

        public enum CommanderSide
        {
            RussianEmpire,
            FrenchEmpire
        };

        public CommanderSide Side { get; }
        public int Health { get; }

        public Commander(CommanderSide side, string id, string name, string description, int health = 0)
        {
            Side = side;
            ID = id;
            Name = name;
            Description = description;
            Health = health;
            ImageFolder = "../../../Data/Images/CommanderImages/" + ID + ".png"; //проверить, написать норм путь
        }

        public override string ToString()
        {
            return Name;
        }
    }
}

File: domain/commander.py
from enum import Enum


class Commander:

    def __init__(self, side_id, id, name, description, health = 0) -> None:
        self.side_id = side_id
        self.id = id
        self.name = name
        self.description = description
        self.health = health
        self.image_folder = "../../../Data/Images/CommanderImages/" + id + ".png" # проверить, написать норм путь

    def __str__(self) -> str:
        return self.name


class CommanderSide(Enum):
    RUSSIAN_EMPIRE = 0
    FRENCH_EMPIRE = 1